runtime/src/lib.rs

```diff
@@ -135,11 +158,11 @@ pub const DAYS: BlockNumber = HOURS * 24;
 // 1 in 4 blocks (on average, not counting collisions) will be primary babe blocks.
 pub const PRIMARY_PROBABILITY: (u64, u64) = (1, 4);
 
-// #[derive(codec::Encode, codec::Decode)]
-// pub enum XcmpMessage<XAccountId, XBalance> {
-//     /// Transfer tokens to the given account from the Parachain account.
-//     TransferToken(XAccountId, XBalance),
-// }
+#[derive(codec::Encode, codec::Decode)]
+pub enum XcmpMessage<XAccountId, XBalance> {
```

Comment:
All looks good. This is the only bit that I wouldn't know what's its use, is it just a leftover from the template that was used as example?